test/testdata/packager/export_imported/a/__package.rb

```diff
@@ -1,12 +0,0 @@
-# frozen_string_literal: true
-# typed: strict
-# enable-packager: true
-
-# Edge case: We technically allow packages to export their imports. It's easier to implement packages this way
-# (although this practice should be discouraged!)
-
-class A < PackageSpec
-  import B
-
-  export B::BClass
-end
```

Comment:
We should almost never delete tests.

Can you change these tests to simply pass, using whatever error assertions are necessary?

Reply:
Ack